frontend/src/Components/UserProfilePosts.tsx

```diff
@@ -99,6 +115,20 @@ export default function UserProfilePosts(props: UserProfilePostsProps) {
           defaultValue={defaultFilter}
         />
       </div>
+      <div className={styles.feedControlsWrapper}>
```

Comment:
the filter (including sorting) should be extracted as a separate component, see e.g. https://github.com/spaceshelter/orbitar/blob/47781e6aa4667d50fada8e7b780a454f88d7bea3/frontend/src/Components/FeedFilterComponent.tsx